book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rental
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
+
+  def add_rental(date, person)
+    Rental.new(date, person, self)
+  end
```

Comment:
- [ ] Good job implementing the requested changes by the previous reviewer. But there is a issue the Rental initializer accepts `date, book, person` as parameters but on this method you are passing `date, person, book`, It will swap the person with the book. Please fix this issue and pass the parameters in this sequence `Rental.new(date, self, person)`